submit/calibJobHandler.py

```diff
@@ -76,7 +76,7 @@
         iters = int(sys.argv[2])
     if ( RunResub ):
         iters = iters + int(sys.argv[2])
-    if ( not RunCRAB and not ONLYHADD and not ONLYFIT and not ONLYFINHADD ):
+    if ( False and (not RunCRAB and not ONLYHADD and not ONLYFIT and not ONLYFINHADD) ):
```

Comment:
@cippy Please remove this False, that was put there to make local tests. Same also at line 224